src/utils/eventsBinder.js

```diff
@@ -1,12 +1,13 @@
 /* vim: set softtabstop=2 shiftwidth=2 expandtab : */
 
-import _ from 'lodash'
+import _ from 'lodash';
 
 export default (vueElement, googleMapObject, events) => {
   _.forEach(events, (eventName) => {
-    const exposedName = eventName;
+    const exposedName = 'g-' + eventName;
```

Comment:
We should really remove the g-. It's not necessary any more since Vue 2 has removed the potential conflict with DOM events